source/_patterns/04-pages/article--research.json

```diff
@@ -1101,6 +1101,36 @@
       },
       "hasAbstracts": false
     },
+    {
+      "bibId": "bib61",
```

Comment:
There are examples of other references in this example page that include a titleLink, what does this add?